### PR TITLE
Issue 84

### PR DESCRIPTION
**Pull request by jdr0887**
This branch fixes three things: 

1. minor bug where some curies were getting formatted like this: "MGI:MGI:<id>"
2. duplicate entries in the Knowledge Graph nodes
3. reverts TRAPI model back to version 0.9.2

### CODE REVIEW

File: src/main/scala/org/renci/cam/QueryService.scala [hunk 1]
```diff
@@ -55,7 +54,13 @@ object QueryService extends LazyLogging {
...
     prefixes
       .filter(entry => value.startsWith(entry._2))
-      .map(entry => s"${entry._1}:" + value.substring(entry._2.length, value.length))
+      .map { entry =>
+        val prefix = s"${entry._1}:"
+        if (value.contains(prefix))
+          value.substring(value.lastIndexOf("/") + 1, value.length)
+        else
+          s"${entry._1}:" + value.substring(entry._2.length, value.length)
+      }
```
Comment (balhoff) [inline src/main/scala/org/renci/cam/QueryService.scala:63]: This seems more complicated than I expect. Applying prefix should find the first entry `(prefix, expansion)` from the map where the value starts with the expansion. Then the expansion should be replaced with the prefix and a colon.

Comment (jdr0887) [reply src/main/scala/org/renci/cam/QueryService.scala:63]: MGI identifiers seem to be delimited by a ":".  We get curies like this: "MGI:MGI:894663" when not checking for `<prefix>:`.

Comment (jdr0887) [reply src/main/scala/org/renci/cam/QueryService.scala]: the following commit uses StringUtils.prefixIfMissing...same behavior, but cleaner

**Comment by jdr0887** [comment]
merged in master & the "test simple query" test runs without issue

**Pull request merged**